src/Analyzers/MSTest.Analyzers.CodeFixes/CodeFixResources.resx

```diff
@@ -135,4 +135,7 @@
   <data name="UseAttributeOnTestMethodFix" xml:space="preserve">
     <value>Add '[TestMethod]'</value>
   </data>
+  <data name="TestClassShouldBeValidFix" xml:space="preserve">
+    <value>Fix '[TestClass]' signature</value>
```

Comment:
I would go with `Fix test class signature` because mentioning `'[TestClass]' signature` feels like we are fixing the attribute